Formula/wine.rb

```diff
@@ -64,53 +51,421 @@ def require_universal_deps?
     sha256 "1b5086798ce6dc959b3cbb8f343ee236ae06c7910e4bbae7d9fde3f162f03a79"
   end
 
-  resource "gecko" do
-    url "https://downloads.sourceforge.net/wine/wine_gecko-2.40-x86.msi", :using => :nounzip
-    sha256 "1a29d17435a52b7663cea6f30a0771f74097962b07031947719bb7b46057d302"
+  resource "gecko-x86" do
+    url "https://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86.msi", :using => :nounzip
+    sha256 "3b8a361f5d63952d21caafd74e849a774994822fb96c5922b01d554f1677643a"
+  end
+
+  resource "gecko-x86_64" do
+    url "https://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86_64.msi", :using => :nounzip
+    sha256 "c565ea25e50ea953937d4ab01299e4306da4a556946327d253ea9b28357e4a7d"
   end
 
   resource "mono" do
-    url "https://downloads.sourceforge.net/wine/wine-mono-4.5.6.msi", :using => :nounzip
-    sha256 "ac681f737f83742d786706529eb85f4bc8d6bdddd8dcdfa9e2e336b71973bc25"
+    url "https://dl.winehq.org/wine/wine-mono/4.6.4/wine-mono-4.6.4.msi", :using => :nounzip
+    sha256 "91b7d58177b9a9355edf007dab94535471aebdddae12734ceb4a219d2ecc4152"
+  end
+
+  resource "openssl" do
```

Comment:
There's no alternative here given the universal situation but my gut feeling is that people will forget to update this `openssl` ASAP when there's a new release. That was the case where we'd vendored crypto stuff before, which isn't always an issue because not everything has `openssl`'s significant & mainstream profile.